src/command_modules/azure-cli-cosmosdb/HISTORY.rst

```diff
@@ -1,5 +1,10 @@
 .. :changelog:
 
+unreleased
+===============
+
+* Minor fix allowing Creation of Collection with custom partition key
```

Comment:
Thanks for point this out. We will update the doc. I'll approve and merge this change.